frontend/src/Components/Endpoint/Endpoint.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "~plaid-threads/scss/variables";
+
+.endpointContainer {
+  display: grid;
+  grid-template-columns: 15% 57% 28%;
+  width: 100%;
+  border: 1px solid $black200;
+  border-bottom: 0px;
+}
+
+.post {
+  margin: 2 * $unit 0px 2 * $unit 3 * $unit;
+  font-size: 1.4rem;
+}
+.endpointContents {
+  margin: 3 * $unit 4 * $unit;
+  font-size: 1.2rem;
+}
+
+.endpointHeader {
+  margin-bottom: $unit;
+}
+
+.endpointName {
+  font-weight: bold;
+  font-size: 2 * $unit;
+  padding-right: $unit;
+}
+
+.schema {
+  font-size: 2 * $unit;
+  font-family: "Inconsolata";
```

Comment:
```suggestion
  font-family: $font-family-monospace;
```

Reply:
again, is there an existing threads style that covers this like the `<code>` style?

Reply:
the <code> style was the wrong font size...